ingestion_tools/scripts/data_validation/tests/annotation/test_segmentationmask_annotation.py

```diff
@@ -0,0 +1,168 @@
+from typing import Dict
+
+import numpy as np
+import pytest
+from mrcfile.mrcinterpreter import MrcInterpreter
+from tests.helper_mrc import HelperTestMRCHeader
+
+ZATTRS_AXIS_ORDER = ["z", "y", "x"]
+
+
+# By setting this scope to session, scope="session" fixtures will be reinitialized for each run + voxel_spacing combination
+@pytest.mark.annotation
+@pytest.mark.parametrize("run_name, voxel_spacing", pytest.run_spacing_combinations, scope="session")
+class TestSegmentationMask(HelperTestMRCHeader):
+
+    @pytest.fixture(autouse=True)
+    def set_class_variables(
+        self,
+        seg_mask_annotation_mrc_headers: Dict[str, MrcInterpreter],
+        voxel_spacing: float,
+        canonical_tomogram_metadata: Dict,
+    ):
+        self.spacegroup = 1  # single 3D volume
+        self.mrc_headers = seg_mask_annotation_mrc_headers
+
```

Comment:
will also probably pull out zarr tests into something similar when working on tiltseries / tomogram validation

Reply:
(this has done in the tiltseries branch)